agent/lib/kontena/load_balancers/configurer.rb

```diff
@@ -14,7 +14,6 @@ class Configurer
     def initialize
       @etcd = Etcd.client(host: self.class.gateway, port: 2379)
       subscribe('lb:ensure_config', :on_ensure_config)
-      subscribe('lb:remove_config', :on_remove_config)
       subscribe('lb:remove_service', :on_remove_service)
```

Comment:
Wait, now the `remove_service` should be cleaned out, and this needs to be `lb:remove_config` instead... The `Kontena::ServicePods::Creator` still sends `lb:remove_config`.